docs/tutorials/HD143006_Part_1.py

```diff
@@ -0,0 +1,211 @@
+
+# reclaiming memory from no-longer-needed variables
+# needed for doc building on github
+# + nbsphinx="hidden"
+# %reset_selective -f dfits
+# %reset_selective -f dnpz
+# %reset_selective -f uu
+# %reset_selective -f vv
+# %reset_selective -f weight
+# %reset_selective -f data
+# %reset_selective -f data_re
+# %reset_selective -f data_im
+# %reset_selective -f fig
+# %reset_selective -f ax
+
+
+def plot(img, imtitle="image"):
+    kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+    fig, ax = plt.subplots(ncols=1)
+    im = ax.imshow(np.squeeze(img), **kw)
+    plt.colorbar(im)
+    ax.set_title(imtitle)
+    ax.set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+    ax.set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+    plt.xlim(left=.75, right=-.75)
+    plt.ylim(bottom=-.75, top=.75)
+
+# img, beam = gridder.get_dirty_image(weighting='uniform')
+# plot(img, imtitle="uniform")
+#
+# img, beam = gridder.get_dirty_image(weighting="briggs", robust=1.0, unit="Jy/arcsec^2")
+# plot(img, imtitle="robust_1.0")
+#
+# img, beam = gridder.get_dirty_image(weighting="briggs", robust=-1.0, unit="Jy/arcsec^2")
+# plot(img, imtitle="robust_-1.0")
+
+img, beam = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/arcsec^2")
+plot(img, imtitle="robust_0")
+
+# Below we plot the DSHARP CLEAN image alongside the MPoL Dirty Image weighted with the Briggs scale and `robust=0.0` for comparison.
+
+kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+fig, ax = plt.subplots(nrows = 2)
+ax[0].imshow(np.squeeze(clean_fits), origin='lower', extent=ext)
+ax[0].set_xlim(left=.75, right=-.75)
+ax[0].set_ylim(bottom=-.75, top=.75)
+ax[0].set_title('DSHARP CLEAN Image')
+ax[0].set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax[0].set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+ax[1].imshow(np.squeeze(img), **kw)
+ax[1].set_title('MPoL Dirty Image')
+ax[1].set_xlim(left=.75, right=-.75)
+ax[1].set_ylim(bottom=-.75, top=.75)
+ax[1].set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax[1].set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+# fig.set_figwidth(14)
+fig.set_figheight(10)
+plt.tight_layout()
+
+# As you can see there are many similarities between the diagnostic dirty image and the image produced by the DSHARP survey using the CLEAN algorithm ([Andrews et al. 2018](https://ui.adsabs.harvard.edu/abs/2018ApJ...869L..41A/abstract)). While the dirty image is more noisy, it still maintains many distinct features present in the CLEAN image. In the next part of the HD143006 tutorial, we will be cleaning the diagnostic dirty image using RML through Neural Networks, Optimization, and Cross Validation with help from [PyTorch](pytorch.org). 
```

Comment:
This paragraph could use an edit. For example, what are the technical differences between the dirty image and the CLEAN image? What do you mean by "distinct features?" 

RML methods are not cleaning methods (those are CLEAN methods), so I wouldn't say that we're cleaning the dirty image using RML. RML and MPoL can take advantage of neural networks, but right now the MPoL package is not a neural network itself.